packages/redux-routine/package.json

```diff
@@ -25,7 +25,7 @@
 	"sideEffects": false,
 	"dependencies": {
 		"@babel/runtime": "^7.9.2",
-		"is-promise": "^2.1.0",
+		"is-promise": "2.1.0",
```

Comment:
Why can't it use `"^2.1.0"`?

Reply:
@gziolo as I was playing around with this PR over the weekend, I have bad luck to pull bad release: https://github.com/then/is-promise/releases/tag/2.2.2, which was later fixed with 3.0.

Reply:
It's now bumped to `^4.0.0`.